integrations/dd-sdk-android-okhttp/src/main/kotlin/com/datadog/android/okhttp/trace/TracingInterceptor.kt

```diff
@@ -882,6 +889,18 @@ internal constructor(
             return getThis()
         }
 
+        /**
+         * Set whether network requests returning a 404 status code should have their resource name redacted.
+         * In order to reduce the cardinality of resource names in APM, 404 urls are automatically redacted to
+         * "404"
+         * @param redacted if true, all 404 requests will have a resource name set to "404", else the resource name
+         * will be the URL
+         */
+        fun set404ResourcesRedacted(redacted: Boolean): R {
```

Comment:
side thought: This API may be confusing for `DatadogInterceptor` users, because this option applies only to APM actually, but they may think that they will see RUM Resource name as "404". But since it is a shared builder, we cannot remove this method from Builder for `DatadogInterceptor`.